apps/studio/tests/integration/helpers/seed/index.ts

```diff
@@ -30,6 +30,35 @@ export const setupAdminPermissions = async ({
     .execute()
 }
 
+export const clearPermissions = async () => {
+  await db.deleteFrom("ResourcePermission").execute()
+}
+
+export const setupPermissions = async ({
```

Comment:
nit - can update `setupAdminPermissions` as a wrapper around this

Reply:
caught me being lazy! i didn't cos it's a test function but i'll do it